3-networks/modules/vpn-ha/README.md

```diff
@@ -0,0 +1,100 @@
+# High Availability VPN module
+
+This module implementes the recomendation proposed in
+[High Availability VPN](https://cloud.google.com/network-connectivity/docs/vpn/concepts/topologies#overview).
+
+## Prerequisites
+
+1. Execution of at least one of the environments in `3-networks/env/`.
+1. Creation on two Cloud Routers for each Region. Both the `Restricted shared VPC`
+and the `Standard shared VPC` modules create two Cloud Routers for each Region
+that should be used as input for this module. **This module only works with two regions.**
+
+## Usage
+
+For each environmnet and for each network, you should add the following peace of code in the
```

Comment:
nit: replace peace with piece or snippet

Reply:
nit: environmnet should also be environment

Reply:
Done!